project3.0/css/style.css

```diff
@@ -1,13 +1,464 @@
+    line-height: 29px;
+}
+
+
+.section-info {
+    display: block;
+    margin-top: 140px;
+}
+
+.container {
+    max-width: 856px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.container-fluid {
+    max-width: 1152px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.section-info-wrapper {
+    display: flex;
+    justify-content: space-between;
+    gap: 115px;
+}
+
+
+.info-left h1 {
+    font-family: var(--font-main);
     font-style: normal;
     font-weight: 700;
     font-size: 44px;
     line-height: 60px;
-    color: #21243D
+    color: var(--color-primary);
+}
+
+.info-left p {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--color-primary);
+}
+
+.info-left button {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 29px;
+    color: #FFFFFF;
+    padding: 13px 20px 13px 20px;
+    background: #FF6464;
+    border-radius: 2px;
+    border-color: #FF6464;
+}
+
+.section-posts {
+    display: block;
+    background: #EDF7FA;
+    margin-top: 71px;
+    padding-bottom: 32px;
+}
+
+.section-posts-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.post-item {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+}
+
+.post-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.post-title h2 {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 60px;
+    color: var(--color-primary);
+}
+
+.post-title a {
+    display: block;
+    padding: 24px 0px 24px 0px;
+    text-decoration: none;
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #00A8CC;
+}
+
+.single-post {
+    background-color: white;
+    max-width: 372px;
+    max-height: 295px;
+    padding-left: 22px;
+    padding-right: 22px;
+}
+
+.single-post h2 {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 26px;
+    line-height: 38px;
+    color: var(--color-primary);
+}
+
+.single-post-date {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 26px;
+    color: var(--color-primary);
+    position: relative;
+    max-width: fit-content;
+}
+
+.single-post p {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--color-primary);
+}
+
+.title-works h2 {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 60px;
+    color: var(--color-primary);
+}
+
+.works-list-item {
+    display: flex;
+    gap: 18px;
+    position: relative;
+}
+
+.works-img {
```

Comment:
First: import images from the figma without border-radius corners.
Second:
Add border-radius here, and overflow:hidden.
And this style:
.works-img > img {
display: block:
width: 100%;
height: auto;
}